blindbid.proto

```diff
@@ -4,10 +4,10 @@ option go_package = "github.com/dusk-network/rusk-schema;rusk";
 
 message GenerateScoreRequest {
     bytes k = 1; // BlsScalar
-    bytes seed = 2; // BlsScalar
+    byte seed = 2; // u8
```

Comment:
The comment is redundant since you're sending a `byte` and not `bytes` (the comments where introduced to let a developer knows which kind of structure expects when `bytes` where sent).